DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -137,3 +138,10 @@
 #miniAOD DQM sequences need to access the filter results.
 PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD)
 PostDQMOffline = cms.Sequence()
+
+
+DQMOfflineMiniAOD = cms.Sequence( miniAODDQMSequence*muonMonitors_miniAOD*MuonMiniAOD )
```

Comment:
please remove this sequence and place the muon miniAOD in the PostDQMOfflineMiniAOD